rbac/management/group/definer.py

```diff
@@ -0,0 +1,88 @@
+#
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""Handler for system defined group."""
+import logging
+
+from django.core.exceptions import ValidationError
+from management.group.model import Group
+from management.policy.model import Policy
+from management.role.model import Role
+from tenant_schemas.utils import tenant_context
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+def seed_group(tenant):
+    """For a tenant create or update default group."""
+    with tenant_context(tenant):
+        name = 'Platform group'
+        group_description = 'Default group that contains default permissions for new users.'
+
+        group, group_created = Group.objects.get_or_create(
+            platform_default=True,
+            defaults={
```

Comment:
Just to clarify, for this `get_or_create`, the `get` will only query for `platform_default=True`, correct? The `defaults` are not a part of the get query? 

Just wanted to confirm, since the `system` flag will not always be `True`.

Reply:
Yes, it will only query for platform_default=True. It will only use the defaults to create.